### DISCUSSION TITLE
RSPduo question

### DISCUSSION THREAD

**Discussion by ah1102** [DSheirer/sdrtrunk#2058] [Q&A]
Can RSPDuo owners say? Do you also see a signal spectrum that goes beyond the strip, as in a screenshot?

![Снимок экрана 2024-10-28 в 16 49 28](https://github.com/user-attachments/assets/d5caff4f-a134-434a-946d-a06c6c235195)

**Comment by DSheirer** [comment]
That is a side-effect of the FFT.  You can try to use a different Window option:

Right click on waterfall: Display >> Window Type

You can try the different Window Type options to see if there is one that reduces the 'splatter'.  However, I've noticed this issue with spectrum from SDRPlay tuners.

**Reply by ah1102** [reply]
I tried to change the type, but the width does not change.

**Reply by ah1102** [reply]
> That is a side-effect of the FFT. You can try to use a different Window option:
> 
> Right click on waterfall: Display >> Window Type
> 
> You can try the different Window Type options to see if there is one that reduces the 'splatter'. However, I've noticed this issue with spectrum from SDRPlay tuners.

Look. The signal width depends on the input signal level.

![Снимок экрана 2024-12-08 в 16 01 27](https://github.com/user-attachments/assets/edd4be06-a8db-42f0-bd5c-6cb2c413d616)

**Comment by ah1102** [comment]
I managed to find the reason. This is in **Display- FFT Width**

FFT Width 512 
![Снимок экрана 2024-11-11 в 21 12 03](https://github.com/user-attachments/assets/5aa8d611-db1f-418d-a62a-5cba93c93b23)

FFT Width 2048
![Снимок экрана 2024-11-11 в 21 06 28](https://github.com/user-attachments/assets/872767d3-7d0d-4975-a361-e02708ef0f8d)

FFT Width 8192
![Снимок экрана 2024-11-11 в 21 11 23](https://github.com/user-attachments/assets/c7ebee8d-6222-4abf-8b37-51b593efc547)
